packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -7772,6 +7772,133 @@ QUnit.module('label integration', () => {
+        assert.strictEqual($multiTagContainer.attr('aria-roledescription'), messageLocalization.format('dxTagBox-tagRoleDescription'), 'aria-roledescription attribute is correct');
+    });
+
+    QUnit.test('root element should have aria-labelledby attribute based from multitag id', function(assert) {
+        const sampleData = ['First Item', 'Second Item', 'Third Item', 'Fourth Item'];
+        const $tagBox = $('#tagBox').dxTagBox({
+            items: sampleData,
+            value: sampleData,
+            maxDisplayedTags: 2,
+            showMultiTagOnly: false,
+        });
+        const $multiTagContainer = $tagBox.find(`.${TAGBOX_MULTI_TAG_CLASS}`);
+        const $parentTagContainerIds = $tagBox.attr('aria-labelledby').split(' ');
+
+        assert.strictEqual($parentTagContainerIds.length, 2, 'root element aria-labelledby attribute contains two ids');
+        assert.strictEqual($parentTagContainerIds.includes($multiTagContainer.attr('id')), true, 'aria-labelledby attribute contains multitag id');
+    });
+
+    QUnit.test('tagBox should show correct count of tags if showMultiTagOnly=true and maxDisplay tags is 2', function(assert) {
+        const sampleData = ['First Item', 'Second Item', 'Third Item', 'Fourth Item'];
+        const $tagBox = $('#tagBox').dxTagBox({
+            items: sampleData,
+            value: sampleData,
+            maxDisplayedTags: 2,
+            showMultiTagOnly: true,
+        });
+        const $parentTagContainerIds = $tagBox.attr('aria-labelledby').split(' ');
+
+        const simpleTagsLength = $tagBox.find(`.${TAGBOX_TAG_CLASS}:not(.${TAGBOX_MULTI_TAG_CLASS})`).length;
+        const multiTagsLength = $tagBox.find(`.${TAGBOX_MULTI_TAG_CLASS}`).length;
+        const $tagIds = $tagBox.find(`.${TAGBOX_TAG_CLASS}`).map((_, element) => $(element).attr('id')).get();
+
+        assert.strictEqual(simpleTagsLength, 0, 'root element has no simple tag');
+        assert.strictEqual(multiTagsLength, 1, 'root element contains one multitag');
+        assert.strictEqual($parentTagContainerIds.every(item => $tagIds.includes(item)), true, 'root element contains all tag ids when showMultiTagOnly = true and maxDisplayTags = 2');
+    });
+
+    QUnit.test('tagBox should show correct count of tags if showMultiTagOnly=true and maxDisplayTags is 4', function(assert) {
+        const sampleData = ['First Item', 'Second Item', 'Third Item', 'Fourth Item'];
+        const $tagBox = $('#tagBox').dxTagBox({
+            items: sampleData,
+            value: sampleData,
+            maxDisplayedTags: 4,
+            showMultiTagOnly: true,
+        });
+        const simpleTagsLength = $tagBox.find(`.${TAGBOX_TAG_CLASS}:not(.${TAGBOX_MULTI_TAG_CLASS})`).length;
+        const multiTagsLength = $tagBox.find(`.${TAGBOX_MULTI_TAG_CLASS}`).length;
+        const $parentTagContainerIds = $tagBox.attr('aria-labelledby').split(' ');
```

Comment:
We use `$` as a prefix only for indicating that variable has a jquery element as a value :) I don't think it's this case.
Check other places please

Reply:
[e4609b8](https://github.com/DevExpress/DevExtreme/pull/27063/commits/e4609b88c463d6b2356659dfd8bec095ea802909)